statuspage/views.py

```diff
@@ -41,4 +44,55 @@ def get_queryset(self):
     def create(self, request, *args, **kwargs):
         request.data['team'] = request.auth.team.pk
         return super().create(request, *args, **kwargs)
-    
\ No newline at end of file
+    
+
+class StatusPageDashboardViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    
+    queryset = StatusPageCategory.objects.all()
+    serializer_class = StatusPageDashboardSerializers
+    permission_classes = []
+    pagination_class = None
+
+    def list(self, request):
+        status_page_config = StatusPageConfiguration.objects.filter(path=self.request.GET.get('path'))
+        if len(status_page_config) == 0:
+            return Response(data={"error": "Please make sure your URL path is exist!"}, status=status.HTTP_404_NOT_FOUND)
+        
+        queryset = StatusPageCategory.objects.filter(team=status_page_config[0].team)
+
+        for category in queryset:
+            success_rate_category = []
+            api_monitor_category = APIMonitorResult.objects \
+                .filter(monitor__team=status_page_config[0].team, monitor__status_page_category=category)
```

Comment:
```suggestion
                .filter(monitor__status_page_category=category)
```
kayaknya filter by category aja cukup fi?

Reply:
Oh tdi sih kepikiran gimana klo ada kemungkinan categorynya sama antar team (gk tau sih bklan kena atau gk wkwk), cuman skrang pathnya udh unique, harusnya udh gk mungkin sih wkwk okee ty